samples/net/sockets/echo_async/README.rst

```diff
@@ -21,7 +20,7 @@ Requirements
 ************
 
 - :ref:`networking_with_qemu`
-- or, a board with hardware networking
+- or, a board with hardware networking (including 6LoWPAN)
```

Comment:
I could read this as you needed a board with 6LoWPAN hardware networking but I think you mean this:

     - or, a board with hardware networking (such as 6LoWPAN)